server/server.js

```diff
@@ -18,36 +18,47 @@ server.use(middlewares);
 server.use(jsonServer.bodyParser);
 server.use(jsonServer.rewriter({
     "/api/documents/popular": "/documents?_sort=popular&_order=desc&_limit=8",
-    "/api/documents?search=:search": "/documents?q=:search",
-    "/api/documents?search=:search&filterId=:filter": "/documents?q=:search&filterId=:filter",
     "/api/*": "/$1"
 }));
 
 
-server.use((req, res, next) => {
+server.use('/documents', (req, res, next) => {
     const { documents } = data;
-    const { q: searchQuery, filtersCount } = req.query;
-    if (typeof filtersCount !== 'string') {
+    const { search, filterId, filtersCount } = req.query;
```

Comment:
the same with boolean and searchStr or searchString for search, I think, would be better.

Reply:
fixed